docs/Maybe.md

```diff
@@ -91,6 +92,13 @@ default value if a `Nothing` is received.
 Produces a pure `Maybe` instance of a given value. Effectively the `Just`
 constructor.
 
+#### `Maybe.toMaybe`
+```hs
+:: a? -> Maybe a
+```
+Produces a `Maybe` instance of a given value. Returns `Nothing` for nil values
+(undefined and null) and `Just(a)` for non-nil values.
```

Comment:
`Just(a)` is a bit confusing, as `a` is a type variable. How about this?

_Alias for `Maybe`. Produces a `Maybe` instance of a given value. Returns `Nothing` for `null`/`undefined`; a `Just` for any other value._